### DISCUSSION TITLE
How does this compare to after effects or in other words: would this be appropriate program for my needs?

### DISCUSSION THREAD

**Discussion by ghost** [friction2d/friction#330] [Q&A]
I like to use flash often (Generally CS6/CS3/8) and wanted to see if I didn't have to use after effects to apply vfx to exported swf or video file. How does this application compare or the intended use case compare to after effects? Would it be a full entire replacement for what after effects can do? Not saying it's trying to be like blender, but be a new tool in flash animation creation.

**Comment by ghost** [comment]
Apologies for the ping, @rodlie, but I know you work on both this project and natron. Is this going to in the future replace natron? Curious to know as well if is or isn't.

**Comment by rodlie** [comment]
> How does this application compare or the intended use case compare to after effects?

Don't know :smile: 

> Would it be a full entire replacement for what after effects can do?

No.

---

Friction is designed for motion graphics, where your target is primarily the web, as in web animations or video. Web animation in this context is SVG.

It might behave/look similar to AE, but that's about it. Friction is not a clone of AE.

> Is this going to in the future replace natron?

Friction and Natron are two completely different applications. Natron is a node based compositing application (clone of Nuke v6), Friction is a layer based vector and raster animation application.

But... if you use Natron for motion graphics you should probably consider Friction.

**Reply by ghost** [reply]
I have used natron, but maybe I wasn't using it right, it would at any given moment just close on me. Is there a specific version that is generally the most stable. I would ask in the natron repo, but there isn't a discussions area so I guess I'll ask since you mentioned using natron instead of friction for motion graphics/vfx.

**Reply by rodlie** [reply]
> but there isn't a discussions area 

Official forum for Natron: https://discuss.pixls.us/c/software/natron

Note that you might need to wait a while for a reply (probably be me), as the Natron community is not that big anymore and the software has been in maintenance mode for many years.

We (Natron) also have a discord, usually more feedback, don't know the url, just search the forum or something.

Regarding stability Natron RB-2.5 is good. Unless you are using Radeon on Linux, then you will need to build it from scratch on Qt5, as we only deliver Qt4 builds (I might fix this before the end of the year.... famous last words).

If you do VFX compositing Natron is for you, else you should consider Friction.
